incubator/kubirds-powerapi-reactor/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: kubirds-telegram-message-reactor
+description: Parametrable Telegram Message Reactor for Kubirds
+type: application
+version: 0.1.0
+appVersion: "1.16.0"
```

Comment:
This must be a copy-pasta from the `kubirds-klander-reactor` chart.

At the time, I did not really understood/care about `version` and `appVersion`. But now I do 😁

So `version` is the chart's version (the templates, values, etc...). Here `0.1.0` is fine.
And `appVersion` is the version of the application being deployed.

By looking at the Docker images:

 - `powerapi/rapl-formula` is `0.4.2`
 - `powerapi/smartwatts-formula` is `0.6.0`

Both Dockerfiles are built from `powerapi/powerapi` but with distinct versions, respectively `0.6.7` and `0.9.3`.

Maybe we should set `appVersion` to the Kubirds version this reactor targets? `1.0.0-rc3` currently?

What do you think?